nengo/__init__.py

```diff
@@ -35,18 +35,22 @@
     )
 del sys
 
+from nengo import spa  # pylint: disable=cyclic-import
+from nengo import dists, exceptions, networks, presets, processes, utils
+
 # Nengo namespace (API)
 from nengo.base import Process
 from nengo.config import Config
 from nengo.connection import Connection
 from nengo.ensemble import Ensemble
-from nengo.node import Node
+from nengo.learning_rules import BCM, PES, RLS, Oja, Voja
+from nengo.network import Network
 from nengo.neurons import (
+    LIF,
```

Comment:
Haha caps sort before lowercase letters so all the all-caps ones go first. Don't think there's more to it than that.

Reply:
That seems like a really weird sorting scheme to sort "L" before "A". That's saying that the second letter in the string is more important than the first letter, which is unlike any sorting scheme I've ever heard of. And even if you do sort that way, then shouldn't "LIFRate" come right after "LIF"?

Edit: figured it out. It's because it thinks "LIF" is a constant (using the heuristic that constants are named using all-caps), whereas "LIFRate" is detected as a class. And it sorts `from` imports by type, so all the "constants" come first, then classes afterwards.